sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -262,7 +262,7 @@ describeFuncName
     ;
 
 describeColName
-    : identifier ('.' (identifier | STRING))*
+    : identifier
```

Comment:
@cloud-fan I changed the syntax here so that the command supports quoted column name but not nested columns. Could you take another look?